src/Message/AuthorizeRequest.php

```diff
@@ -74,6 +74,14 @@ class AuthorizeRequest extends AbstractRequest
 {
     const API_VERSION_STATEMENT_DESCRIPTOR = "2014-12-17";
 
+    /**
+     * @return mixed
+     */
+    public function getTransferData()
+    {
+        return $this->getParameter('transferData');
```

Comment:
Ditto. Keep this the same as before and make the driver translate to this.

Reply:
done